docs/ignore.md

```diff
@@ -13,13 +13,13 @@ To exclude files from formatting, create a `.prettierignore` file in the root of
 
 Example:
 
-```
```

Comment:
What's the point of marking it as `text`?

Reply:
To satisfy https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md040---fenced-code-blocks-should-have-a-language-specified, I can remove it.

Reply:
I just was curious.

Reply:
I'll keep it then. I'll need update docs in #13530